pkg/translator/ingress_utils.go

```diff
@@ -47,6 +48,8 @@ type ingressOwnedEdgeLBObjectMetadata struct {
 	Namespace string
 	// IngressBackend is the reconstructed IngressBackend object represented by the current EdgeLB object (in case said object is an EdgeLB backend).
 	IngressBackend *extsv1beta1.IngressBackend
+	// Protocol is either http or https
+	Protocol string
```

Comment:
Can we add a type for this?

```
type Protocol string

const (
	HTTP Protocol = "http"
	HTTPS Protocol = "https"
)
```

Or something like that?

Reply:
This is a bigger code change. I'll do it in a follow up PR.